charts/rstudio/templates/fluentd-configmap.yml

```diff
@@ -8,33 +8,39 @@ metadata:
     release: "{{ .Release.Name }}"
   name: "{{ template "fullname" . }}-fluentd"
 data:
-  elasticsearch_host: "{{ .Values.Elasticsearch.Host }}"
-  elasticsearch_port: "{{ .Values.Elasticsearch.Port }}"
   fluent.conf: |
     <source>
-      type tail
-      format none
+      @type tail
+      format /^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*) +\S*)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")? "(?<x_forwarded_for>[^ ]+)?"/
+      time_format %d/%b/%Y:%H:%M:%S %z
       path /var/log/rstudio-server/rserver-http-access.log
-      pos_file /var/log/rstudio-server/rserver-http-access.pos
-      read_from_head true
-      tag rstudio-access
+      tag "#{ENV['USER']}.rstudio.access"
     </source>
 
     <source>
-      type tail
-      format none
+      @type tail
       path /var/log/rstudio-server/rserver-http-error.log
-      pos_file /var/log/rstudio-server/rserver-http-error.pos
-      read_from_head true
-      tag rstudio-error
+      format none
+      tag "#{ENV['USER']}.rstudio.error"
     </source>
 
+    # Ignore k8s healthcheck requests
+    <filter *.nginx.access>
+      @type grep
+      exclude1 agent ^Go-http-client\/.*$
```

Comment:
As discussed offline, from what I can tell in this case the healtcheck endpoint is `/health-check`, so we may prefer to filter by `path` field:

```ruby
exclude1 path ^\/health-check$
```